cli/tests/services/touch_file/check_activation_starts_services.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+echo "activating"
+exec 3>&- # close the file descriptor
+eval $("$FLOX_BIN" activate --start-services)
+sleep 1 # wait for the services to start
+
+echo "stopping process-compose"
+"$PROCESS_COMPOSE_BIN" down -u "${_FLOX_SERVICES_SOCKET}" || true
```

Comment:
Nit: the timeout and trap approach that we originally used for Python, and I've copied to https://github.com/flox/flox/pull/1763, saves a bit of time when the socket appears in <1s and is more robust about the cleanup. I expect the tests will change a lot in the coming week(s) though anyway.